doordeck-sdk/src/mingwMain/resources/doordeck_headless_sdk/converter/AuthKeyResponseConverter.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System.Text.Json;
 using System.Text.Json.Serialization;
-using DoordeckHeadlessSDK.model.responses;
+using Doordeck.Headless.Sdk.model.responses;
```

Comment:
I guess whilst we're looking at naming, should `model` be `Model`, etc? 

https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names
- Use PascalCase for class names and method names.
- Use camelCase for method parameters and local variables.
- Use PascalCase for constant names, both fields and local constants.

Reply:
I also did it for the Data/Responses, added some default values to avoid NPE warnings.

Although I noticed some issues while testing it, I will take a look before merging